ocaml/xapi/xapi_globs.ml

```diff
@@ -1614,6 +1616,11 @@ let other_options =
     , (fun () -> string_of_bool !disable_webserver)
     , "Disable the host webserver"
     )
+  ; ( "request-worker-pool-size"
+    , Arg.Set_int request_worker_pool_size
+    , (fun () -> string_of_int !request_worker_pool_size)
+    , "Specify size of worker pool servicing xapi requests, must be non-zero."
```

Comment:
Maybe `worker thread pool` ?
Why not 0 ? I mean, won't that get the old no pool behaviour ?
Also, what if a negative number is specified ?